lib/lookup/src/lookup_v2/jit.rs

```diff
@@ -0,0 +1,216 @@
+use crate::lookup_v2::{BorrowedSegment, Path};
+use std::str::CharIndices;
+
+#[derive(Clone)]
+pub struct JitPath<'a> {
+    path: &'a str,
+}
```

Comment:
Should this also be `Copy`?

Reply:
It could, but it's not used. `Path` only required `Clone` to allow implements that aren't necessarily `Copy` only.